package.json

```diff
@@ -39,37 +38,45 @@
         "react-gtm-module": "^2.0.11",
         "react-keybind": "^0.9.4",
         "react-mde": "^11.5.0",
-        "react-monaco-editor": "^0.33.0",
+        "react-monaco-editor": "^0.54.0",
         "react-router-dom": "^5.3.4",
         "react-select": "^5.2.2",
         "react-toastify": "^8.2.0",
-        "react-virtualized": "^9.22.3",
+        "react-virtualized": "^9.22.5",
         "recharts": "^2.1.9",
+        "rollup-plugin-node-polyfills": "0.2.1",
         "rxjs": "^7.5.4",
+        "sockjs-client": "1.6.1",
         "tippy.js": "^6.3.7",
+        "vite": "^4.5.0",
+        "vite-plugin-require-transform": "1.0.21",
+        "vite-plugin-svgr": "^2.4.0",
+        "vite-tsconfig-paths": "^4.2.0",
         "xterm": "^4.19.0",
         "xterm-addon-fit": "^0.5.0",
         "xterm-addon-search": "^0.9.0",
         "xterm-webfont": "^2.0.0",
-        "yaml": "^1.7.2",
-        "yamljs": "^0.3.0"
+        "yaml": "^1.7.2"
     },
     "scripts": {
         "prepare": "husky install",
-        "lint": "tsc --noEmit && eslint 'src/**/*.{js,jsx,ts,tsx}' --max-warnings 0",
+        "lint": "tsc --noEmit && NODE_OPTIONS=--max_old_space_size=8192 eslint 'src/**/*.{js,jsx,ts,tsx}' --max-warnings 0",
         "lint-fix": "eslint 'src/**/*.{js,jsx,ts,tsx}' --fix",
-        "start": "react-app-rewired start",
-        "build": "react-app-rewired build",
-        "build-light": "GENERATE_SOURCEMAP=false react-app-rewired build",
-        "build-k8s-app": "GENERATE_SOURCEMAP=false REACT_APP_K8S_CLIENT=true react-app-rewired build",
-        "test": "react-app-rewired test",
+        "start": "vite",
+        "build": "tsc && NODE_OPTIONS=--max_old_space_size=4096 vite build",
```

Comment:
Do we need tsc here? We can add the lint check before build instead and get rid of this since the typing is not required for web app